spring-cloud-task-stream/src/test/java/org/springframework/cloud/task/launcher/TaskLauncherSinkTests.java

```diff
@@ -0,0 +1,86 @@
+package org.springframework.cloud.task.launcher;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.deployer.spi.task.LaunchState;
+import org.springframework.cloud.stream.annotation.Bindings;
+import org.springframework.cloud.stream.messaging.Sink;
+import org.springframework.cloud.task.launcher.configuration.TaskConfiguration;
+import org.springframework.cloud.task.launcher.util.TaskLauncherSinkApplication;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.messaging.support.GenericMessage;
+
+public class TaskLauncherSinkTests {
+
+	private final static String DEFAULT_STATUS = "test_status";
+
+	private AnnotationConfigApplicationContext context;
+
+	@Before
+	public void setUp() {
+		context = new AnnotationConfigApplicationContext();
+		context.setId("testTask");
+	}
+
+	@After
+	public void tearDown(){
+		if(context != null){
+			context.close();
+		}
+	}
+
+	@Autowired
+	@Bindings(TaskLauncherSink.class)
+	private Sink sink;
+
+	@Test
+	public void testSuccess() {
+		context.register(TaskLauncherSinkApplication.class, TaskConfiguration.class);
+		context.refresh();
```

Comment:
Any reason to do this in the tests instead of using SpringRunner and `@ConfigurationContext`?